http-transport-binding.md

```diff
@@ -163,16 +163,23 @@ Except for attributes [explicitly handled in this specification]
 HTTP header mapping of well-known CloudEvents attributes is that 
 each attribute name MUST be prefixed with "CE-".
 
+Note: per the [HTTP](https://tools.ietf.org/html/rfc7230#section-3.2)
+specification, header names are case-insensitive. Upper-case letters are
+prefixed with a dash ("-"). When converting a HTTP Header Name back, the dash
+converts the following letter back to upper-case, all other letters are
+lower-case.
+
 Examples:
 
-    * `eventTime` maps to `CE-EventTime`
-    * `eventID` maps to `CE-EventID`
-    * `cloudEventsVersion` maps to `CE-CloudEventsVersion`
+    * `eventTime` maps to `CE-Event-Time`
```

Comment:
`CE-Event-Time` is now more inline with the HTTP Convention (e.g. `Content-Type` is also separated by a dash) than the previous `CE-EventTime.

Reply:
Someone mentioned on the call the redundancy of the word "event", does it make things a bit cleaner to omit that from properties?
https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html

Reply:
i.e:  `time` maps to `ce-time`

Reply:
if we did this, we should probably do it for all properties that have the word "event" in them - like EventID.

However, I kind of like having it there even if it does seem redundant in some cases. Mainly because it avoids the risk of it overlapping with some other property.  When we talked about future properties or extensions we always talked about how people should pick good name to avoid potential conflicts. To me that means avoiding really generic terms that could be used for other purposes. I think "Time" would fall into that category since there could be many different "times" added as properties. Keeping "Event" in there (to me) scopes it to a CloudEvent property - where we've (hopefully) clearly defined its semantics/meaning.

Reply:
While I think it is a valid concern that can be discussed, I'm not sure if this PR is the best place to do so - @PerryBirch what do you think about opening a separate issue (or PR) instead?

I think this PR can be approved or rejected independently of that.